code/modules/power/supermatter/supermatter.dm

```diff
@@ -844,21 +878,21 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		var/mob/living/user = AM
 		if(user.status_flags & GODMODE)
 			return
-		message_admins("[src] has consumed [key_name_admin(user)] [ADMIN_JMP(src)].")
-		investigate_log("has consumed [key_name(user)].", INVESTIGATE_ENGINES)
+		var/add
+		if(user.mind?.assigned_role == "Clown")
```

Comment:
user?.mind?.assigned_role

A body could be thrown in without any mind, causing that to runtime